plugins/inputs/http_listener_v2/http_listener_v2.go

```diff
@@ -36,7 +36,8 @@ type TimeFunc func() time.Time
 // HTTPListenerV2 is an input plugin that collects external metrics sent via HTTP
 type HTTPListenerV2 struct {
 	ServiceAddress string            `toml:"service_address"`
-	Path           string            `toml:"path"`
```

Comment:
Again, you need to keep this and add it to your `paths` list to preserve backward compatibility.